controllers/application.go

```diff
@@ -298,3 +300,45 @@ func (c *ApiController) DeleteApplication() {
 	c.Data["json"] = wrapActionResponse(object.DeleteApplication(&application))
 	c.ServeJSON()
 }
+
+func (c *ApiController) validateApplicationURLs(application object.Application) {
```

Comment:
move to map where key - field, value - error 
than iterate over map and call IsURLValid

Reply:
done